### ISSUE TITLE
Create peripheral with no local config file

### ISSUE THREAD

**Issue by lptr** [cornucopia-machines/ugly-duckling-firmware#221]
Looks like we haven't been initializing peripherals when config files went missing:

```text
2024-10-05 15:14:31.524 [Error] Failed to create 'flow-control' peripheral 'flow-control-a' because ConfigurationException: Cannot open config file /p/flow-control-a (EmptyInput)
```

The peripheral shows up on the admin interface, though, which is further misleading.

**Comment by lptr** [comment]
Looks like the `EmptyInput` comes from here https://github.com/kivancsikert/ugly-duckling/blob/63785b3c7091dade66e34af819b5bfa8f6bf4a8b/main/kernel/Configuration.hpp#L301

So it is a JSON parsing problem.

I tried to get the file causing the above problem via the `files/read` command, and I got a strange response:

```json
{"path":"/p/flow-control-a","size":93,"contents":""}
```

So the file-system says the file has 93 bytes of content, but when reading, it is somehow "empty". The API used in reading the file is `Stream.readString()` which uses a timed-read, i.e. it can return an empty string if the read times out. But why would reading from the file system time out?

Or is the string somehow garbled instead, and gets read as an empty string instead? What would cause such garbling?

For reference, here's the `FileRead` command:

https://github.com/kivancsikert/ugly-duckling/blob/63785b3c7091dade66e34af819b5bfa8f6bf4a8b/main/kernel/Command.hpp#L130-L146
